app/Http/routes.php

```diff
@@ -362,6 +362,13 @@
     'as' => 'comment',
 ]);
 
+Route::delete('comment/{commentId}', 'CommentController@deleteComment');
```

Comment:
**Code Functionality **

Please ensure there is an auth middleware for this route to ensure only authenticated users can delete their comment